dataprofiler/profilers/profile_builder.py

```diff
@@ -80,16 +81,31 @@ def __init__(self, df_series, sample_size=None, min_sample_size=5000,
             'data_stats_profile':
                 ColumnStatsProfileCompiler(clean_sampled_df, self.options)}
 
-        # use the data labeler by default
-        use_data_labeler = True
-        if options and isinstance(options, StructuredOptions):
-            use_data_labeler = options.data_labeler.is_enabled
-
-        if use_data_labeler:
+        data_labeler = self.options.data_labeler.data_labeler_model
```

Comment:
This isn't just the `data_labeler_model`, the data labeler contains a pre/post processor and a model. Recommend renaming to `data_labeler`, also why are there two levels to this? 

`data_labeler.data_labeler`?